packages/pwa-buildpack/lib/queries/index.js

```diff
@@ -0,0 +1,12 @@
+// Execute this first so we can import .graphql files directly.
+const requireGraphQL = require('require-graphql-file');
+
+// Import all the build-time queries.
+const getMediaUrl = requireGraphQL('../queries/getStoreMediaUrl');
+const getSchemaTypes = requireGraphQL('../queries/getSchemaTypes');
+
+// Export the queries for use by the rest of buildpack.
+module.exports = {
+    getMediaUrl,
+    getSchemaTypes
+};
```

Comment:
This file isn't _strictly_ necessary, and we'll have to add all new `.graphql` files here to make them available for the rest of `buildpack`, but we do need to `require` the `.graphql` files in a different way (using `requireGraphQL`) so this consolidates that logic.

Reply:
I would prefer that. Having a simple index.js makes things simpler for the future.